scripts/HFIR_4Circle_Reduction/mpl2dgraphicsview.py

```diff
@@ -8,17 +8,15 @@
 from __future__ import (absolute_import, division, print_function)
 import os
 import numpy as np
-
-from PyQt4 import QtGui
-
+from qtpy.QtWidgets import (QWidget, QVBoxLayout, QSizePolicy)
```

Comment:
There is an abstraction in the root of `scripts` to make matplotlib qt4/qt5 compatible. Use that.